include/cograph_recognition/CographRecognition.hpp

```diff
@@ -0,0 +1,68 @@
+
+
+#pragma once
+
+#include <optional>
+#include <vector>
+
+#include <networkit/base/Algorithm.hpp>
+#include <networkit/components/ConnectedComponents.hpp>
+#include <networkit/graph/Graph.hpp>
+#include <networkit/graph/GraphTools.hpp>
+
+namespace Koala {
+
+
+class CoraphRecognition : public NetworKit::Algorithm {
+ public:
+    
+    enum class State {
+            UNKNOWN,
+            COMPLEMENT_REDUCIBLE,
+            COND_1,
+            COND_2,
+            COND_3,
+            COND_4,
+            COND_5,
+            COND_6
```

Comment:
What is this doing here? And why is there no "NOT_COMPLEMENT_REDUCIBLE"? (when we are sure it's not a cograph)

In general COGRAPH, NOT_COGRAPH is better, one does not have to infer that "CO" is a shorthand for complement-reducibility.